python/acquire/__init__.py

```diff
@@ -56,11 +62,14 @@ def setup_one_streams(runtime: Runtime, frame_count: int) -> Properties:
     ]
     logging.warning(f"Cameras {cameras}")
 
+    if len(cameras) == 0:
+        cameras = ["simulated.*sin.*"]
+
     p.video[0].camera.identifier = dm.select(DeviceKind.Camera, cameras[0])
     p.video[0].storage.identifier = dm.select(DeviceKind.Storage, "Trash")
     # p.video[0].storage.settings.filename = output_filename
     p.video[0].camera.settings.binning = 1
-    p.video[0].camera.settings.shape = (2304, 2304)
+    p.video[0].camera.settings.shape = (64, 64)
```

Comment:
I arbitrarily picked a small simulated radial sine image as a quick way to test things out. Don't have a strong preference about what the simulated image should be here.